src/bin/pg_autoctl/ipaddr.c

```diff
@@ -594,3 +594,95 @@ findHostnameFromLocalIpAddress(char *localIpAddress, char *hostname, int size)
 
 	return true;
 }
+
+
+/*
+ * resolveHostnameForwardAndReverse returns true when we could do a forward DNS
+ * lookup for the hostname, and then a reverse DNS lookup, and the reverse DNS
+ * lookup returns a single IP address that is the same as the forward lookup
+ * returned.
```

Comment:
> reverse DNS lookup returns a single IP 

a reverse dns lookup returns a hostname right? Can you double check this comment being correct?

Reply:
That was more like a draft, thanks for catching it. Reworded now.